### ISSUE TITLE
Hello, fileRename not working

### ISSUE THREAD

**Issue by ghost** [multitheftauto/mtasa-blue#847]
Hello, I opened a topic in the mta forum and asked for support for a code, but the code does not work on my computer, it works only on the members who posted it.

fileRename can not rename the file, even the code working for other members, I've tried several things and the same error always happens.

The fileExists returns: true, then the file is in the right place, plus the fileRename can not rename the file.

Even with code working perfectly for others, this error always happens, has already been done all kinds, reinstallation, permissions.:
https://i.imgur.com/Nu3EZiT.png

```
function onStartResourceDeleteFiles()
    if fileRename("filetxt/asd.txt","filetxt/asd2.txt") then
        print("File `filetxt/asd.txt` successfully renamed to `filetxt/asd2.txt`")
    else
        print("Unable to rename `filetxt/asd.txt`")
		print(fileExists("filetxt/asd.txt"))
    end 
end
addEventHandler("onClientResourceStart", getResourceRootElement(getThisResource()), onStartResourceDeleteFiles)
```

https://forum.mtasa.com/topic/115189-ajuda-o-que-est%C3%A1-de-errado-nesse-c%C3%B3digo/

Please, if you have any PT BR support, it would be better to understand.

**Comment by ghost** [comment]
The problem was corrected in the version: [mtasa-1.5.6-rc-16571-20190307.exe﻿](https://nightly.mtasa.com/mtasa-1.5.6-rc-16571-20190307.exe)

**Comment by qaisjp** [comment]
Fixed in the following commits: 3a7c803936d54636c58aec677d99cbbac68a4ff3, e9ce827def7b4dc5dcd97f3ac4e8fd22c3b234ea
